### ISSUE TITLE
`neo4j:constraints` and `neo4j:indexes` not working

### ISSUE THREAD

**Issue by ProGM** [neo4jrb/neo4j-rake_tasks#33]
### Code example (inline, gist, or repo)

```
$ bin/rake neo4j:constraints[development]
Getting Neo4j constraints for development...
rake aborted!
EOFError: end of file reached

Tasks: TOP => neo4j:constraints
(See full trace by running task with --trace)
```

### Runtime information:

Neo4j database version:
`neo4j-rake_tasks` gem version:  0.7.0
`neo4j` database version: 2.3.3

**Comment by cheerfulstoic** [comment]
I just released some fixes, but I wasn't getting quite the same error as you.  Could you try with 0.7.1 and if that doesn't work share the output of the rake task with `--trace`?

**Comment by ProGM** [comment]
@cheerfulstoic 
After upgrading to 0.7.1, it seems to be fixed! ;)
However this task is not showing constraints on relations. Any plans for it?

**Comment by cheerfulstoic** [comment]
Good point!  Released as 0.7.2!
